ravedude/src/board.rs

```diff
@@ -316,6 +317,31 @@ impl Board for SparkFunProMicro {
     }
 }
 
+struct SparkFunProMini5V;
+
+impl Board for SparkFunProMini5V {
+    fn display_name(&self) -> &str {
+        "SparkFun Pro Mini 5V (16MHz)"
+    }
+
+    fn needs_reset(&self) -> Option<&str> {
+        None
+    }
+
+    fn avrdude_options(&self) -> avrdude::AvrdudeOptions {
+        avrdude::AvrdudeOptions {
+            programmer: "arduino",
+            partno: "atmega328p",
+            baudrate: Some(57600),
+            do_chip_erase: true,
+        }
+    }
+
+    fn guess_port(&self) -> Option<anyhow::Result<std::path::PathBuf>> {
+        Some(find_port_from_vid_pid_list(&[(0x0403, 0x6001)]))
```

Comment:
We can't assume that any old FTDI chip is a sparkfun pro mini 5v.  The `guess_port` should only be populated when there is a unique VID/PID that identifies this board without any doubt.

```suggestion
        Some(Err(anyhow::anyhow!("Not able to guess port")))
```